docs/UserGuide.md

```diff
@@ -74,7 +80,7 @@ faster than traditional GUI apps.
 </div>
 
 ## Contact Management
-Contacts represent candidates applying for job positions. Each contact can have details such as name, phone, email, and relevant skills, aiding in recruitment processes.
+Contacts represent candidates applying for job positions. Each contact can have details such as name, phone, email, and relevant skills, aiding in recruitment processes. A contact's role represents the job that they are applying for, and is used when screening contacts with a certain job. A contact is uniquely identified by their phone number and email, therefore no two contacts can share a same phone number or email. 
```

Comment:
near the last line, no two contacts can share THE same phone number or email.

Reply:
Fixed!